src/entity_embedding_utils.jl

```diff
@@ -125,7 +127,8 @@ function MLJModelInterface.transform(
     fitresult,
     Xnew,
 )
-    is_embedding_enabled_type(transformer) || return Xnew
+    # if it doesn't have the property its not an entity-enabled model
+    hasproperty(transformer, :embedding_dims) || return Xnew
     ordinal_mappings, embedding_matrices = fitresult[3:4]
     Xnew = ordinal_encoder_transform(Xnew, ordinal_mappings)
     Xnew_transf = embedding_transform(Xnew, embedding_matrices)
```

Comment:
Also, do we have any test for `transform(::MLJFluxModel, ...)` ?

Reply:
I think `transform` is tested in `entity_embedding.jl`

Reply:
Okay, I see it's covered in Codecov. 


Can I ask that this method (and corresponding test) be moved to "mlj_model_interface.jl", where the other implementations of `MLJModelInterface` methods (such as `fit` and `predict`) live?